### DISCUSSION TITLE
Basamaklı sayı sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#203] [Soru Cevap]
MN ve KL iki basamaklı iki sayıyı, MMM de üç basamaklı bir sayıyı ifade etmektedir. Bu üç sayı arasında MN + KL = MMM ilişkisi olduğuna göre K kaçtır?

**Comment by Sadik1Dev** [comment]
MN=10M+N
KL=10K+L
MMM=111M
10M+N+10K+L=111M
10K+L+N=101M
KL+N=101M
9>=N>0

buradan yola çıkarsak KL sayısının N tek basamaklı sayısı ile toplandığında en az 3 basamaklı bir sayıya dönüşmesi için K'in alabileceği tek değer 9 dur.
K=9 çünkü onlar basamağı 9 olmayan iki basamaklı bir sayı bir rakamla toplanınca 3 basamaklı bir sayı olamaz.
